### ISSUE TITLE
about the result

### ISSUE THREAD

**Issue by SJTUzhanglj** [orobix/Prototypical-Networks-for-Few-shot-Learning-PyTorch#5]
I run your code with default settings, but get very pool results of accurcy (about 53%). What could be the reason for this?

**Comment by dnlcrl** [comment]
Hi @SJTUzhanglj, I'm not sure about what you are doing wrong but those are my results for the first 5 epochs when I clone and run the code with the default parameters.

**Comment by dnlcrl** [comment]
Hi @SJTUzhanglj, I'm not sure about what you are doing wrong but those are my results for the first 5 epochs when I clone and run the code with the default parameters with cuda enabled (using --cuda).

=== Epoch: 0 ===
100%|█████████████████████████████████████████| 100/100 [00:04<00:00, 24.92it/s]
Avg Train Loss: 0.5905507113039493, Avg Train Acc: 0.8372999995946884
Avg Val Loss: 0.04801010820905503, Avg Val Acc: 0.9844000053405761 (Best)
=== Epoch: 1 ===
100%|█████████████████████████████████████████| 100/100 [00:03<00:00, 25.17it/s]
Avg Train Loss: 0.24386439725756645, Avg Train Acc: 0.9284666675329208
Avg Val Loss: 0.029075963221921484, Avg Val Acc: 0.9898666727542877 (Best)
=== Epoch: 2 ===
100%|█████████████████████████████████████████| 100/100 [00:03<00:00, 25.15it/s]
Avg Train Loss: 0.17279222041368483, Avg Train Acc: 0.9506666612625122
Avg Val Loss: 0.019735890002148152, Avg Val Acc: 0.9938666707277298 (Best)
=== Epoch: 3 ===
100%|█████████████████████████████████████████| 100/100 [00:03<00:00, 25.15it/s]
Avg Train Loss: 0.14480093855410814, Avg Train Acc: 0.9567666608095169
Avg Val Loss: 0.022845985335152363, Avg Val Acc: 0.9930666702985763 (Best: 0.9938666707277298)
=== Epoch: 4 ===
100%|█████████████████████████████████████████| 100/100 [00:03<00:00, 25.17it/s]
Avg Train Loss: 0.13216185364872218, Avg Train Acc: 0.960733328461647
Avg Val Loss: 0.01483979538510198, Avg Val Acc: 0.9950666701793671 (Best)
=== Epoch: 5 ===
100%|█████████████████████████████████████████| 100/100 [00:03<00:00, 25.13it/s]
Avg Train Loss: 0.12260673478245736, Avg Train Acc: 0.9627666652202607
Avg Val Loss: 0.014596411682811094, Avg Val Acc: 0.9950666701793671 (Best)

As you can see I got 99.5% accuracy on the validation set at the 5th epoch. Also, almost the same results are obtained without using cuda (after the 1st epoch I got: Avg Train Acc: 0.8359333354234696, Avg Val Acc: 0.9846666723489761), so all I can say is to try to pull/clone again the code and re-run the training. 

I am closing the issue, but feel free to re-open it if you still need help or want to add some info.
